inst/tests/tests.Rraw

```diff
@@ -6921,10 +6921,23 @@ ll = sapply(ll, paste, collapse=",")
 test(1477.07, transpose(strsplit(ll, ",", fixed=TRUE)), tstrsplit(ll, ",", fixed=TRUE))
 test(1477.08, transpose(1:5), error="l must be a list")
 test(1477.09, transpose(list(as.complex(c(1, 1+5i)))), error="Unsupported column type")
-test(1477.10, transpose(list(list(1:5))), error="Item 1 of list input is")
+test(1477.10, transpose(list(x~y)), error="Item 1 of list input is") # changed behavior since we also support now list columns #5805
```

Comment:
In 5 years, this comment will make no sense. I would just remove it. It would have been useful as a PR comment or note in the PR description.

```suggestion
test(1477.10, transpose(list(x~y)), error="Item 1 of list input is")
```